vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -17,6 +17,18 @@ KUBERNETES_VERSION: "{{ lookup('env', 'KUBERNETES_VERSION') | default('v1.18.0',
 KUBERNETES_BINARIES_VERSION: "{{ lookup('env', 'KUBERNETES_BINARIES_VERSION') | default(lookup('env', 'KUBERNETES_VERSION') | default('v1.18.0', true), true) }}"
 KUBERNETES_BINARIES_CONFIG_VERSION: "{{ lookup('env', 'KUBERNETES_BINARIES_CONFIG_VERSION') | default('v0.2.7', true) }}"
 
+IP_STACK: "{{ lookup('env', 'IP_STACK') | default('v4', true) }}"
+PROVISIONING_POOL_RANGE_START: "{{ lookup('env', 'PROVISIONING_POOL_RANGE_START') | default('172.22.0.100', true) }}"
+PROVISIONING_POOL_RANGE_END: "{{ lookup('env', 'PROVISIONING_POOL_RANGE_END') | default('172.22.0.200', true) }}"
+PROVISIONING_CIDR: "{{ lookup('env', 'PROVISIONING_CIDR') | default('24', true) }}"
+BAREMETALV4_POOL_RANGE_START: "{{ lookup('env', 'BAREMETALV4_POOL_RANGE_START') | default('192.168.111.100', true) }}"
+BAREMETALV4_POOL_RANGE_END: "{{ lookup('env', 'BAREMETALV4_POOL_RANGE_END') | default('192.168.111.100', true) }}"
```

Comment:
Should BAREMETALV4_POOL_RANGE_START and BAREMETALV4_POOL_RANGE_END be both defaulted default('192.168.111.100', true)?

Reply:
fixing it